controller/aks-cluster-config-handler.go

```diff
@@ -498,386 +479,132 @@ func BuildUpstreamClusterState(ctx context.Context, secretsCache wranglerv1.Secr
 
 // updateUpstreamClusterState compares the upstream spec with the config spec, then updates the upstream AKS cluster to
 // match the config spec. Function returns after a update is finished.
-func updateUpstreamClusterState(ctx context.Context, secretsCache wranglerv1.SecretCache, spec aksv1.AKSClusterConfigSpec, upstreamSpec *aksv1.AKSClusterConfigSpec) (*aksv1.AKSClusterConfigSpec, error) {
-	if spec.Tags == nil {
-		spec.Tags = map[string]string{}
-	}
-	spec.Tags["displayName"] = spec.ClusterName
-	if spec.Imported || reflect.DeepEqual(spec.Tags, upstreamSpec.Tags) &&
-		spec.NodePools != nil && reflect.DeepEqual(spec.NodePools, upstreamSpec.NodePools) {
-		return nil, nil
-	}
```

Comment:
Does this mean that before this change, you couldn't update anything besides tags and node pools?

Reply:
Well, AKS doesn't offer much options to change. Unfortunately It was limited at the beginning, but it was updated now

Reply:
Could you mention that in the commit message?

Reply:
Added to commit message